src/pager/pager.tsx

```diff
@@ -165,6 +164,7 @@ export default class Pager extends PureComponent<PagerProps> {
   }
 
   getPagerLinks() {
+    const messages = this.context as Messages;
```

Comment:
In order not to duplicate the type assertions we can do it the following way. This is the officially recommended way of typing the context in class components.
```ts
static contextType = I18nContext;
declare context: React.ContextType<typeof I18nContext>;

const messages = this.context;
```

Reply:
We can even do this:
```ts
static contextType = I18nContext;
declare context: React.ContextType<typeof Pager.contextType>;
```

Reply:
Addressed in https://github.com/JetBrains/ring-ui/pull/5542